database/src/main/scala/cromwell/database/migration/metadata/table/executionevent/ExecutionEventTableDescriptionMigration.scala

```diff
@@ -0,0 +1,37 @@
+package cromwell.database.migration.metadata.table.executionevent
+
+import cromwell.database.migration.metadata.MetadataCustomSql
+
+class ExecutionEventTableDescriptionMigration extends MetadataCustomSql {
+
+  override def queries: Array[String] = {
+    Array(
+      """
+        |INSERT INTO METADATA_JOURNAL (
+        |  WORKFLOW_EXECUTION_UUID,
+        |  METADATA_KEY,
+        |  CALL_FQN,
+        |  JOB_SCATTER_INDEX,
+        |  JOB_RETRY_ATTEMPT,
+        |  METADATA_VALUE,
+        |  METADATA_VALUE_TYPE,
+        |  METADATA_TIMESTAMP
+        |)
+        |  SELECT
+        |    WORKFLOW_EXECUTION_UUID,
+        |    CONCAT('executionEvents[', ev.EVENT_ID ,']:description'),
```

Comment:
The index here is a synthetic PK in the EXECUTION_EVENTS table, does that make sense here?